### PR TITLE
Model Evaluation GA - Updating model cards to add evaluation support

### PR DESCRIPTION
**Pull request by sarthaks95**

### CODE REVIEW

File: assets/models/system/bert-base-cased/spec.yaml [hunk 1]
```diff
@@ -65,4 +67,53 @@ tags:
...
+      Standard_NC24s_v3,
+      Standard_NC64as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96asr_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND40rs_v2,
+    ]
+version: 14
```
[the remark this answers is not in the document]
Comment (sarthaks95) [reply assets/models/system/bert-base-cased/spec.yaml]: Downgraded.

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 2]
```diff
@@ -4,6 +4,8 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 19308160448536e378e3db21a73a751579ee7fdd
+  evaluation-min-sku-spec: 16|0|64|128
```
[the remark this answers is not in the document]
Comment (sarthaks95) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: It's needed for ES for validation.

File: assets/models/system/bert-base-cased/spec.yaml [hunk 3]
```diff
@@ -65,4 +67,53 @@ tags:
...
+      Standard_NC16as_T4_v3,
+      Standard_NC24s_v3,
+      Standard_NC64as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96asr_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND40rs_v2,
+    ]
```
[the remark this answers is not in the document]
Comment (sarthaks95) [reply assets/models/system/bert-base-cased/spec.yaml:118]: Done

**Review by BhavanaTumma — DISMISSED** [review]
Approved

**Review by BhavanaTumma — APPROVED** [review]
Approved

**Pull request merged**